Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -395,7 +395,7 @@ public struct ConvertAction: Action, RecreatingContext {
         }
 
         // Log the output size.
-        benchmark(add: Benchmark.OutputSize(dataURL: targetDirectory.appendingPathComponent(NodeURLGenerator.Path.dataFolderName)))
```

Comment:
This seems reasonable to me. Does this removal have an effect if the `--transform-for-static-hosting` flag isn't included in the benchmark run?

Reply:
It will depend a bit on the benchmark run. At the very least we should see a slight bump for the new `index/index.json` file since that's emitted by default now.

And then if we're also building the LMDB version of the index or building with a DocC-Render template, we'll see an even larger increase.

I'm running a benchmark now that includes DocC-Render template so I'm expecting a noticeable bump from that.